src/main/java/uk/go/hm/icb/dto/ICBMultiMatch.java

```diff
@@ -17,4 +17,5 @@ public class ICBMultiMatch {
     private String birthCertificate;
     private String passportNumber;
     private String flag;
+    private String fileName;
```

Comment:
Do we need the fileName in multi match? I think this may only be available with single record search result.